OD_handler/OD_handler.py

```diff
@@ -0,0 +1,464 @@
+    Returned variables:
+    flags
+	'''
+
+	#Initialize the argument parser
+	parser = argparse.ArgumentParser()
+
+	#Adding
+	parser.add_argument("-e", "--estimations", help="Get only the estimations for every sample", action = "store_true")
+	parser.add_argument("-f", "--figures", help="Get only the growth curve figures", action = "store_true")
+	parser.add_argument("-s", "--summary", help="Get only the summary of growth rate estimations",action = "store_true")
+	parser.add_argument("-b", "--bioshaker", help="Get one growth rates figures for every individual bioshaker", action = "store_true")
+	parser.add_argument("-i", "--individual", help="Get one growth rates figures for every individual sample", action = "store_true")
+	parser.parse_args()
+	args = parser.parse_args(argv_list[1:])
+
+	#Create flags 
+
+	if args.estimations == False and args.figures == False and args.individual == False and  args.summary == False :
+		
+		flag_all = True
+		flag_est = False
+		flag_sum = False
+		flag_fig = False
+		flag_ind = False
+		flag_bioshaker = False
+
+	elif args.estimations == False and args.figures == False and args.individual == True and  args.summary == False :
+
+		flag_all = True
+		flag_est = False
+		flag_sum = False
+		flag_fig = False
+		flag_ind = True
+		flag_bioshaker = False
+
+	else :
+
+		flag_all = False
+		flag_est = args.estimations
+		flag_sum = args.summary
+		flag_fig = args.figures
+		flag_ind = args.individual
+		flag_bioshaker = args.bioshaker
+
+	return flag_all, flag_est, flag_sum, flag_fig, flag_ind, flag_bioshaker
+
+
+
+# ------ PARSE THE DATA USING THE autoflow_parser LIBRARY ------
+
+
+def parse_data() :
+	'''Calls the autoflow_parser and returns a merged xlsx document with all the OD readings combined'''
+	try :
+		call = subprocess.call("autoflow_parser",shell = True)
+	except :
+		return sys.exit("The data could not be parsed due to some error, check the input documentation")
+	return call
+
+
+
+# ------ DATA LOADING AND VARIABLE SELECTION ------
+
+
+def read_xlsx(filename = "results.xlsx") : #done
+	'''Reads .xlsx file, returns a dataframe with relevant variables. The output of the parser is set to be "results.xlsx", the default reads the mentioned file without any additional argument'''
+	try :
+		#Open data file
+		df = pd.read_excel(filename)
+	except FileNotFoundError :
+		return sys.exit("Could not find the parsed data file (XLSX extension)")
+	#Select valuable columns
+	cols = ["Sample ID", "Measurement", "Measurement type", "Sampling time","Sampling date"]	#relevant variables
+	df = df[cols]
+	df.columns = df.columns.str.replace(' ', '_')	#header spaces replaced with underscore
+	
+	return df
+
+
+
+# ------ SEPARATION OF GROWTH RATE SAMPLE AND VOLUME LOSS SAMPLES ------
+
+
+def sample_outcome(sample_file, df) : #done
+	'''Uses an external file containing individual sample purposes, returns two classifed dataframes based on sample purposes and labelled by bioshaker.
+	Keyword arguments:
```

Comment:
Thanks for adding the arguments to the docstrings.  This makes it faster to understand the code.